### PR TITLE
Splashscreen: change variants order / fix msg "You did not set any plugins, parser, or stringifier"

### PR DESCRIPTION
**Pull request by deraswer**
## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/swaponline/MultiCurrencyWallet/blob/master/docs/CONTRIBUTING.md) guide
- [ ] Good naming, keep simple
- [ ] Tested desktop/mobile
- [ ] Tested bright/dark
- [ ] Tested en/ru
- [ ] Affects money; I checked the functionality once again
- [x] I checked the PR once again

### Original issue

#3825

### Video / screenshot proof

image

image

### CODE REVIEW

File: webpack/rules/scss.js [hunk 1]
```diff
@@ -31,6 +31,7 @@ export default [
           postcssOptions: {
             plugins: () => [
               autoprefixer(AUTOPREFIXER),
+              require('postcss-import')
```
Comment (NotEternal) [inline webpack/rules/scss.js:34]: I set the same string, but problem does not disappear)

Comment (deraswer) [reply webpack/rules/scss.js:34]: idk what`s the problem, but for me it works as in the attached screenshot. maybe try to restart your de

Comment (NotEternal) [reply webpack/rules/scss.js:34]: ok, I'll check it again

Comment (NotEternal) [reply webpack/rules/scss.js:34]: terminal

I was checking in the terminal (without code editors). First off, deleted **node_modules**, but the same mistake

**Review by shendel — APPROVED** [review]

File: webpack/rules/scss.js (hunk 1 above)
Comment (deraswer) [reply webpack/rules/scss.js:34]: checked this also, and you are right, even without this line after caching (after first build) this message disappears, so this change was useless after all

Comment (noxonsu) [inline webpack/rules/scss.js:34]:
```suggestion
              require('postcss-import')
```

File: src/front/client/index.html [hunk 2]
```diff
@@ -828,8 +828,8 @@ <h2 class="starter-modal__title">
       <script>
         const btnCreateWallet = document.querySelector('.starter-modal__btn_create-wallet');
         const btnConnectWallet = document.querySelector('.starter-modal__btn_connect-wallet');
-        const btnSkip = document.querySelector('.starter-modal__btn_skip');
-        const btnRestoryWallet = document.getElementById('starter-modal__link-restore-wallet')
+        const btnSkip = document.querySelector('.starter-modal__link_skip');
+        const btnRestoryWallet = document.getElementById('starter-modal__btn-restore-wallet');
```
Comment (NotEternal) [inline src/front/client/index.html:832]:
```suggestion
        const btnRestoryWallet = document.getElementById('starter-modal__btn-restore-wallet');
        // ...
        // Restore wallet
        btnRestoryWallet.addEventListener('click', () => {
           redirectionTo('#/restoreWallet'); // добавить эту строку
           setCookie('swapDisalbeStarter', 'true', {
            expires: SEC_IN_YEAR * 5,
          });
        })
```

**Pull request closed without merging**